test/hca_metadata_api/test.py

```diff
@@ -85,6 +95,7 @@ def _rename_keys(self, d, **kwargs):
                 d[new_name] = d[old_name]
                 del d[old_name]
 
+    @valid_schema_domains(['schema.dev.data.humancellatlas.org'])
```

Comment:
This is a marginal aspect of our tests and I don't think I want to pollute this many test methods with a decorator invocation for the sake of each test having a minimal set of valid domains. We should provide this patch in a super class of our unit tests case classes and just allow the union of all observed domains there.